test/test_meta.py

```diff
@@ -876,6 +876,19 @@ def test_empty_quantized(self):
         r = torch.empty(2 ** 52, device='meta', dtype=torch.qint8)
         self.assertEqual(r.device.type, 'meta')
 
+    def test_map_location_deserialize(self):
+        import io
+
+        t = torch.rand(10)
+        b = io.BytesIO()
+
+        torch.save(t, b)
+        b.seek(0)
+        r = torch.load(b, map_location=torch.device("meta"))
```

Comment:
Can we add an assertion that `(r.storage().nbytes())`  < `t.storage().nbytes()`

Reply:
no, because this assert is not true, but we can assert that r.data_ptr() is null